### PR TITLE
Fix some grammar issues.

### PR DESCRIPTION
**Pull request by rowild**
Additionally to grammar fixes:
- Capitalize titles.
- Remove irrelevant text.

Questions:
"This procedure" => better: "This command"?
"[...] to use the Introduction with TYPO3 CMS 9.5 [...]" => "[...] to use the Introduction **Package** with TYPO3 CMS 9.5 [...]"?

This sentence does not make any sense:
"[...]`composer-typo3-auto-install` additionally takes care of the initial setup if
it's not already done, means no LocalConfiguration.php can be found. [...]"

I guess you mean:
"[...]`composer-typo3-auto-install` additionally takes care of the initial setup in case 
no `LocalConfiguration.php` can be found. [...]"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Get going quickly with TYPO3 CMS Introduction Package and DDEV Local.
 
-This repository provides a fully setup and ready to use TYPO3 CMS instance
+This repository provides a full setup and ready to use TYPO3 CMS instance
```
Comment (gilbertsoft) [inline README.md]: fully is correct here imho but set up should be written with two words instead.

**Review by gilbertsoft — CHANGES_REQUESTED** [review]
Thanks for your suggestions, very appreciated!

**Comment by gilbertsoft** [comment]
> "This procedure" => better: "This command"?
> "[...] to use the Introduction with TYPO3 CMS 9.5 [...]" => "[...] to use the Introduction **Package** with TYPO3 CMS 9.5 [...]"?

This has some historical reason, in the first version I only described a procedure to do so. You can change it to command if you like, imho both is right.

> This sentence does not make any sense:
> "[...]`composer-typo3-auto-install` additionally takes care of the initial setup if
> it's not already done, means no LocalConfiguration.php can be found. [...]"
> 
> I guess you mean:
> "[...]`composer-typo3-auto-install` additionally takes care of the initial setup in case
> no `LocalConfiguration.php` can be found. [...]"

I'm fine with your suggestion here, go ahead.

File: README.md (hunk 1 above)
Comment (rowild) [reply README.md]: It is correct if "set up" is a verb, true. Do you want me to adapt that change?

Comment (rowild) [reply README.md]: Will do :-)

**Review by gilbertsoft — APPROVED** [review]

**Pull request merged**